mlvtools/conf/conf.py

```diff
@@ -5,7 +5,9 @@
 from os.path import join, exists, basename, dirname
 from typing import List
 
+import yaml
 from pydantic import BaseModel, validator, ValidationError
+from yaml import YAMLError
```

Comment:
Really not a big deal, but I think you can skip this redundant import (since you have yaml imported on line 8)

Reply:
Thks, I will change it :)